helm/conjur-config-cluster-prep/templates/clusterrole.yaml

```diff
@@ -1,3 +1,4 @@
+# This ClusterRole defines the Kubernetes API access permissions that the Conjur authenticator will require in order to validate application identities.
```

Comment:
Would you mind splitting up all of the comments into 80-character (or thereabouts) line lengths (preceding each line in the block comment with a `#`)? The line length you have is valid YAML, but in all of our other code types, we generally keep max line length to about 80 chars.